### ISSUE TITLE
make back button behaviour better and improve ux

### ISSUE THREAD

**Issue by ash23090** [truefedex/tv-bro#49]
make back button behaviour like open browser available in android tv play store,pressing back will give options like going forward,back,add to bookmark,go to home,etc with big icons just like open browser/seraphric browser,thanks.

**Comment by killerz298** [comment]
I agree with this request if there is a simple way to implement it. I'm still using the original Shield remote which isn't great but serves it's purpose but button options are limited. Long press or multi-press functionality would certainly help with that. I've been debating replacing the remote anyway.

**Comment by ashwin-2309** [comment]
__
